scalding-core/src/main/scala/com/twitter/scalding/serialization/StringOrderedSerialization.scala

```diff
@@ -22,7 +22,67 @@ import scala.util.control.NonFatal
 
 import JavaStreamEnrichments._
 
+object StringOrderedSerialization {
+  final def binaryIntCompare(leftSize: Int, seekingLeft: InputStream, rightSize: Int, seekingRight: InputStream): Int = {
```

Comment:
This is moving a function you already had which returned int, though it suits my purposes that it returns Int so it matches all the standard compare functions.